manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114/WorkFlowy.WorkFlowy.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114
+PackageLocale: en-US
+Publisher: WorkFlowy
+PackageName: WorkFlowy
+License: Proprietary
+Copyright: Copyright © 2023 WorkFlowy
+ShortDescription: Minimilist-design inifinite bullet-list outliner
```

```suggestion
ShortDescription: Workflowy offers a simpler way to stay organized. If you have a crazy job or an ambitious project, we will be your trusty sidekick.
```
There are typos in the ShortDescription. I would suggest sticking to this one from the official website.